docs/policy/policy_create_hubs.rst

```diff
@@ -0,0 +1,14 @@
+#Policy considerations while creating a new hub
+
+We have mastered creating new hubs based on our prior experience creating 10+ hubs catering to the diverse needs of the instructors across the campus. Our decisions with regards to creating a new hub were made with a lot of intuition around solving instructors' needs effectively within the time constraints.
+The objective of this policy document is to verbalize our heuristics for creating a new hub so that it can guide the decision making of all team members in the future. 
+
+- Course has a large user base with more than 300+ students (Eg: Data 8 and Data 100 hubs)
```

Comment:
We should order these in order of importance, and basically say 'keep going down the list, and the sooner you hit a condition that is satisfied, the more likely you should get your own hub'.

Just to re-arrange this a bit, I think it should be:

1. Course needs image customization beyond what we currently have in the main datahub image. It doesn't necessarily need to be incompatible, but more than 'just this one package' - we don't want complexity. This is most likely the primary reason
2. Admin access
3. Testbed for deploying new features.
4. Computationaly intensive + calendar based scheduling.
5. Organizational reasons